CSharpOOP4/Ball.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Kristina_Kulich__Application.CSharpOOP4;
+
+public class Ball : Shape
+{
+    private double _radius;
+    
+    public Ball(string name, double radius):base(name)
+    {
+        this._radius = radius;
+    }
+
+    public override void ShapeInfo()
+    {
+        Console.WriteLine("Area of {0}   = {1:F2}",Name, this.GetSquare());
```

Comment:
answer you question, you can use property Name from basic class or here you can call base realization of method from basic class without creating public property.(base.ShapeInfo();)